params/config.go

```diff
@@ -536,6 +536,13 @@ func (c *ChainConfig) IsTerminalPoWBlock(parentTotalDiff *big.Int, totalDiff *bi
 	return parentTotalDiff.Cmp(c.TerminalTotalDifficulty) < 0 && totalDiff.Cmp(c.TerminalTotalDifficulty) >= 0
 }
 
+// IsParis returns whether the given block is either equal to the Paris (merge)
+// fork or greater.
+// Note: Only usable for post-merge networks where MergeNetsplitBlock has been configured.
```

Comment:
Wait... 

- returns whether the given block is either `>= merge`,
- only usable for `> merge` -networks

??

Reply:
Ah never mind, I get it